packages/dashboard/src/components/MultiCountChart/examples/MultiCountChart.Example.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { AnnotationType, MultiCountChart } from '../index';
+
+export class MultiCountChartExample extends React.Component<{}, {}> {
+  public render(): JSX.Element {
+    const rows = [
+      {
+        data: 13000,
+        bodyText: 'Flagged users',
+        annotaionText: 'Annotation Text',
+        color: 'blue',
+        type: AnnotationType.positive
+      },
+      {
+        data: 8000000,
+        bodyText: 'Risky sign-ins',
+        annotaionText: 'Decrease in the safety',
+        color: 'green',
+        type: AnnotationType.negative
+      },
+      {
+        data: 331100000,
+        bodyText: 'Risky sign-ins',
+        annotaionText: 'Annotation',
+        color: 'red',
+        type: AnnotationType.nuetral
```

Comment:
neutral*

Reply:
Good question. It is as shown above. The `bodyText` string is what gets the ellipses, not the `annotationText`. The example is on the last frame in the Figma file.